src/main/java/mate/academy/spring/validator/Email.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.validator;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = EmailValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Email {
```

Comment:
https://github.com/mate-academy/jv-spring-rest-web-practice#:~:text=!%20Don%27t%20create%20GlobalExceptionHandler%2C%20custom%20validation%20annotations%20in%20this%20HW%20(this%20is%20not%20specified%20in%20the%20task%20condition)